elections/steering/2025/README.md

```diff
@@ -0,0 +1,316 @@
+does not count towards the final count.
+
+[Eligible voters] may endorse candidates of their choosing by replying to the
+candidate's nomination **issue** saying something like "I endorse this nominee,
+and I work for <COMPANY>" or "+1". Please state that you an eligible voter,
+and include your employer's name so that we see can which candidates have
+sufficient endorsements.
+
+Note that **only endorsements on the GitHub issue will be considered**.
+Endorsements on the nomination email will NOT be considered.
+
+When a candidate has reached the necessary three endorsements, one of the
+Election Officers will announce that on the GitHub issue.
+
+**Running**
+
+Eligible candidates can submit a pull request with a biography in this
+directory with their platform and intent to run. This PR will copy `nomination-template.md`
+to a file named `candidate-yourgithub.md`.  It will fill out the fields in
+that template.
+
+All biographical statements should be brief and to the point, with a guideline of around
+**300 words** total for all text. If your statement is excessively long, you
+will be asked to abbreviate it before it is merged.
+
+Please refer to the [prior candidate bios] for examples of content.
+Biography statements are optional.
+
+Missed deadlines by the candidates will be addressed by the election committee
+on a per case basis to determine eligibility.
+
+**Campaigning**
+
+Please refer to the [Steering Committee Election Charter] and understand
+that we care deeply about [limiting corporate campaigning]. The election
+officers and members of the steering committee [pledge to recuse] themselves
+from any form of electioneering.
+
+You should be running as a "brand free" individual, based on your contribution
+to the project as a member of this community, outside of whatever corporate
+roles you may hold.
+
+## Voting Process
+
+See the [eligibility section](#eligibility) to understand if you are eligible to vote.
+
+Elections will be held using time-limited [Condorcet] ranking on [Elekto].
+The most preferred candidates will be elected to the open seats.
+
+Employer diversity is encouraged, and thus maximal representation will be
+enforced as spelled out in the [Steering Committee Election Charter].
+
+You will be ranking your choices of the candidates with an option for
+"no opinion". In the event of a tie, a non-involved SC member will flip
+a coin.
+
+The election will open for voting starting on the dates specified on the calendar
+at the top of this document. You will be reminded that voting has opened by an
+email to kubernetes-dev, but no email is required for you to vote.
+
+### Officers
+
+The Steering Committee has selected the following people as [election officers]:
+
+- Christoph Blecker
+- Nina Polshakova
+- Sreeram Venkitesh
+
+In addition, the following contributors are helping with the election:
+
+- Alternate Officers: Rey Lejano, Arujjwal Negi
+- Infra Liaison: TBD
+- Contributor Comms Liaison: TBD
```

Comment:
What about these roles?

Reply:
again, selected by the officers after they are seated.